### ISSUE TITLE
Removing Media Folder from Magento Site

### ISSUE THREAD

**Issue by fabwebstudio** [arkadedigital/magento-s3#15]
After sync media folder to 'S3', is it possible we can remove the 'Media' folder from Magento site?
Seems magento checking if file does exist on server then fetching it from S3.

Please let me know if that's possible, that way we can save some space on server.

**Comment by thaiphan** [comment]
Hey @fabwebstudio,

This extension takes advantage of Magento's built-in "database" media storage system to work. This system copies stuff to the file system as a sort of caching mechanism. There's nothing we can do about that. Sorry!

If you configure the Magento media URL to use S3 or CloudFront, you'll find that Magento will download remote images into the media folder less often.

Regards,

Thai

**Comment by thaiphan** [comment]
Hey @fabwebstudio,

Can I close this issue?

Regards,

Thai

**Comment by fabwebstudio** [comment]
Yeah, close that.. I have figured that out :), by mounting media repo to
server directly.

*Best Regards*
*Arun Verma*
Director, Web/Mobile App Development
av@fabwebstudio.com
Skype: fabwebstudio

This transmission is confidential and may be legally privileged. It is
intended only for the stated addressee(s) and access to it by any other
person is unauthorised. If you are not the intended recipient, you must not
disclose, copy, circulate or in any other way use or rely on the
information contained in this email. If you have received this email in
error, please notify the sender by return e-mail and/or av@fabwebstudio.com and
delete the message from your system.

**Comment by thaiphan** [comment]
Hey @fabwebstudio,

Okay

**Comment by thaiphan** [comment]
Hey @fabwebstudio,

Okay thanks! Are you still using this extension? Would you be interested in providing a success story?

Regards,

Thai
